From eb46058b1b0272f17c29614687196de0f040aa01 Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Mon, 23 Sep 2013 14:23:48 +0800
Subject: [PDK KK Temp] disable linker warning log

BZ: 149708

disable it temporarily as we need to enable EAT test.

Change-Id: I0fbcf89b92023093864869beebf034e9a73a5f79
Category: temp enablement
Domain: AOSP-bionic
Origin: internal
Upstream-Candidate: no
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 linker/linker.cpp |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 56480b4..24509b6 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -1532,8 +1532,8 @@ static bool soinfo_link_image(soinfo* si) {
          * phdr_table_protect_segments() after all of them are applied
          * and all constructors are run.
          */
-        DL_WARN("%s has text relocations. This is wasting memory and is "
-                "a security risk. Please fix.", si->name);
+       //  DL_WARN("%s has text relocations. This is wasting memory and is "
+       //         "a security risk. Please fix.", si->name);
         if (phdr_table_unprotect_segments(si->phdr, si->phnum, si->load_bias) < 0) {
             DL_ERR("can't unprotect loadable segments for \"%s\": %s",
                    si->name, strerror(errno));
-- 
1.7.4.1

